installer.sh

```diff
@@ -29,44 +22,36 @@ source ./.scripts/util.sh
 # For Windows, check that we are running in a MSYS2 UCRT environment
 check_env
 
-PACKAGE_INSTALL_HELP="\
-#  Installs required library packages.
-#  ATM only MSWEBVIEW2 on Windows is fully supported, so we leave this as basic logic.
-#  The installer currently supports CEF 
-#  @todo Expand the logic for multiple OS and embedded browser packages.
+
+SELECT_BROWSER_HELP="\
+#  Selects the browser engine to use
+## Options:
+#      Chromium      Use Chromium Embedded Framework (CEF)
+#      MSWebView2    Use Microsoft WebView2 (Windows only)
 ## Usage:
-#   packages_install\
+#      select_browser <chromium|mswebview2>                               (required)
 "
-
-select_browser_caller() {
+select_browser() {
     if [ "$(is_help "$@")" = true ]; then
         print_help "$SELECT_BROWSER_HELP"
         [ $PS1 ] && return 0 || exit 0
     fi
     if [[ -z "$1" ]]; then
-        throw_error "Browser type must be specified. Use: Chromium or MSWebView2"
+        throw_error "Browser type must be specified: $BROWSER_OPTIONS"
         return 1
     fi
     source ./.scripts/util.sh
-    select_browser "$1"
+    select_browser_caller "$1"
 
 }
 
 PACKAGE_INSTALL_HELP="\
 #  Installs required library packages.
-#  Supports both CEF (Chromium) and MS WebView2 browser engines.
-#  Browser must be selected first using select_browser command.
+#  ALPHA stage: Supports MsWebview2 and Chromium on Windows OS.
+
 ## Usage:
 #   packages_install\
 "
-SELECT_BROWSER_HELP="\
-#  Selects the browser engine to use
-## Options:
-#      Chromium      Use Chromium Embedded Framework (CEF)
-#      MSWebView2    Use Microsoft WebView2 (Windows only)
-## Usage:
-#      select_browser <chromium|mswebview2>                               (required)
-"
 
 packages_install() {
```

Comment:
Add the logic `if [ -w $BROWSER ]; then select_browser fi` so that the default is selected if the user did not specify.

Reply:
Add this logic to `generate` as well. A user may have already installed packages in a previous terminal session. Now they do `source installer.sh` again, but the environment will have forgotten about the `BROWSER`, so it needs to be set again.